docs/add-ins/toc.yml

```diff
@@ -2,11 +2,13 @@
   href: index.md
 - name: Office Add-ins platform overview
   href: /office/dev/add-ins/overview/office-add-ins?context=outlook/context
-- name: Get started
+- name: Quick Starts
```

Comment:
Quick starts

Reply:
@lindalu-MSFT should this TOC entry be "Quick Starts" or "Quick starts" ?  I modeled this after office-js-docs-pr doc set, which uses "Quick Starts" -- if that's wrong, then I'll update in both places. Please let me know your thoughts. Thanks!

Reply:
Definitely Quick Starts with capitalization for consistency. It also corresponds with the headers on the product portals that have Quick Start on the header.

Reply:
I noticed that the many of the other TOC entries have sentence casing. Should they be updated as well?

Reply:
It's my understanding that other TOC entries are correct as-is; Quick Starts is an exception to the rule, for consistency with how that term appears on other MSFT web properties like the Dev Portal. (@lindalu-MSFT please feel free to chime in if you have anything else to add.)